### PR TITLE
Issue serialization

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

1. Minor edits
 2. links included

### CODE REVIEW

File: serialization.md [hunk 1]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
+Its a common scenario to use Flink to process Pravega Stream data produced by a non-Flink application.  The Pravega client library used by such applications defines the [`io.pravega.client.stream.Serializer`](http://pravega.io/docs/javadoc/v0.1.0/clients/io/pravega/client/stream/Serializer.html) interface for working with event data. The implementations of `Serializer` directly in a Flink program via built-in adapters can be used:
+- [`io.pravega.connectors.flink.serialization.PravegaSerializationSchema`](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/main/java/io/pravega/connectors/flink/serialization/PravegaSerializationSchema.java)
+- [`io.pravega.connectors.flink.serialization.PravegaDeserializationSchema`](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/main/java/io/pravega/connectors/flink/serialization/PravegaDeserializationSchema.java)
 
-Pass an instance of the appropriate Pravega de/serializer class to the adapter's constructor.  For example:
+Below is an example, to pass an instance of the appropriate Pravega de/serializer class to the adapter's constructor:
```
Comment (Aparnarr) [inline serialization.md:34]: @vijay  The below statement should be only in the last line???? or can be moved to top line!..?

Note that the Pravega serializer must implement `java.io.Serializable` to be usable in a Flink program.

File: serialization.md [hunk 2]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
 DataStream<MyEvent> stream = env.addSource(reader);
 ```
 
-### Interoperability with Other Applications
-A common scenario is using Flink to process Pravega stream data produced by a non-Flink application.  The Pravega client library used by such an application defines the [`io.pravega.client.stream.Serializer`](http://pravega.io/docs/javadoc/v0.1.0/clients/io/pravega/client/stream/Serializer.html) interface for working with event data.  You may use implementations of `Serializer` directly in a Flink program via built-in adapters:
-- `io.pravega.connectors.flink.serialization.PravegaSerializationSchema`
-- `io.pravega.connectors.flink.serialization.PravegaDeserializationSchema`
+## Interoperability with Other Applications
+Its a common scenario to use Flink to process Pravega Stream data produced by a non-Flink application.  The Pravega client library used by such applications defines the [`io.pravega.client.stream.Serializer`](http://pravega.io/docs/javadoc/v0.1.0/clients/io/pravega/client/stream/Serializer.html) interface for working with event data. The implementations of `Serializer` directly in a Flink program via built-in adapters can be used:
```
Comment (vijikarthi) [inline serialization.md]: Please change the Pravega link to http://pravega.io/docs/latest/javadoc/clients/io/pravega/client/stream/Serializer.html

File: serialization.md [hunk 3]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
     http://www.apache.org/licenses/LICENSE-2.0
 -->
+# Serialization
 
-_Serialization_ refers to converting a data element in your Flink program to/from a message in a Pravega stream.
+ Converting a data element in your Flink program to/from a message in a Pravega Stream is referred as **Serialization**.
 
-Flink defines a standard interface for data serialization to/from byte messages delivered by various connectors.  The core interfaces are:
+Flink defines a standard interface for data serialization to/from byte messages delivered by various connectors. The core interfaces are:
 - [`org.apache.flink.streaming.util.serialization.SerializationSchema`](https://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java/org/apache/flink/streaming/util/serialization/SerializationSchema.html)
```
Comment (vijikarthi) [inline serialization.md]: Please change link to https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/streaming/util/serialization/SerializationSchema.html

File: serialization.md [hunk 4]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
 - [`org.apache.flink.streaming.util.serialization.DeserializationSchema`](https://ci.apache.org/projects/flink/flink-docs-release-1.3/api/java/org/apache/flink/streaming/util/serialization/DeserializationSchema.html)
```
Comment (vijikarthi) [inline serialization.md]: Please change the link to https://ci.apache.org/projects/flink/flink-docs-stable/api/java/org/apache/flink/streaming/util/serialization/DeserializationSchema.html

File: serialization.md [hunk 5]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
-- `io.pravega.connectors.flink.serialization.PravegaSerializationSchema`
-- `io.pravega.connectors.flink.serialization.PravegaDeserializationSchema`
+## Interoperability with Other Applications
+Its a common scenario to use Flink to process Pravega Stream data produced by a non-Flink application.  The Pravega client library used by such applications defines the [`io.pravega.client.stream.Serializer`](http://pravega.io/docs/javadoc/v0.1.0/clients/io/pravega/client/stream/Serializer.html) interface for working with event data. The implementations of `Serializer` directly in a Flink program via built-in adapters can be used:
+- [`io.pravega.connectors.flink.serialization.PravegaSerializationSchema`](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/main/java/io/pravega/connectors/flink/serialization/PravegaSerializationSchema.java)
```
Comment (vijikarthi) [inline serialization.md]: Please change the link to reflect the master branch
https://github.com/pravega/flink-connectors/blob/master/src/main/java/io/pravega/connectors/flink/serialization/PravegaSerializationSchema.java

File: serialization.md [hunk 6]
```diff
@@ -7,30 +7,31 @@ You may obtain a copy of the License at
...
+- [`io.pravega.connectors.flink.serialization.PravegaDeserializationSchema`](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/main/java/io/pravega/connectors/flink/serialization/PravegaDeserializationSchema.java)
```
Comment (vijikarthi) [inline serialization.md]: Please change the link to reflect master branch

https://github.com/pravega/flink-connectors/blob/master/src/main/java/io/pravega/connectors/flink/serialization/PravegaDeserializationSchema.java

File: serialization.md (hunk 1 above)
Comment (vijikarthi) [reply serialization.md:34]: Let it be there as-is i.e in the `Interoperability with Other Applications` section.

**Review by vijikarthi — CHANGES_REQUESTED** [review]

File: serialization.md (hunk 1 above)
Comment (Aparnarr) [reply serialization.md:34]: ok

File: serialization.md (hunk 2 above)
Comment (Aparnarr) [reply serialization.md]: done

File: serialization.md (hunk 6 above)
Comment (Aparnarr) [reply serialization.md]: done

File: serialization.md (hunk 5 above)
Comment (Aparnarr) [reply serialization.md]: done

File: serialization.md (hunk 4 above)
Comment (Aparnarr) [reply serialization.md]: done

File: serialization.md (hunk 3 above)
Comment (Aparnarr) [reply serialization.md]: done

**Review by vijikarthi — APPROVED** [review]

**Pull request merged**